src/java/org/apache/cassandra/repair/autorepair/FixedSplitTokenRangeSplitter.java

```diff
@@ -117,20 +107,24 @@ private KeyspaceRepairAssignments getRepairAssignmentsForKeyspace(boolean primar
 
         if (byKeyspace)
         {
+            long totalBytes = repairPlan.getEstimatedBytes();
+            long bytesPerRange = Math.max(1, totalBytes / splitsPerRange);
```

Comment:
Approximating bytes per range as a function of `totalBytes / splitsPerRange` can work, but it could also be pretty inaccurate if there are wide partitions.  I think its fine but probably just need to document it.